HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -24,6 +24,13 @@ def esproducers_by_type(process, *types):
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+def customiseFor14282(process):
```

Comment:
This will not work online as online does not use this file!
We need to plan a proper migration incl. ConfDB parsing.